librephotos.env

```diff
@@ -0,0 +1,71 @@
+# This file contains all the things you need to change to set up your Libre Photos. 
+# There are a few items that must be set for it to work such as the location of your photos.
+# After the mandatory entry's there are some optional ones that you may set. 
+
+# Start of mandatory changes. 
+
+# Location of your photos.
+myPhotos=./librephotos/pictures
+
+# Comma delimited list of patterns to ignore (e.g. "@eaDir,#recycle" for synology devices)
+skipPatterns=
+
+# The location where you would like to store the log files. The full path must exist as it will not be created for you. 
+logLocation=./librephotos/logs/
+
+# Protected media directory. This is where we store some files like the thumbnails of your images. 
+proMedia=./librephotos/media
+
+# Where shall we store the backend and frontend code files. 
+codedir=./librephotos/code
```

Comment:
This is an optional parameter and should be placed at the bottom.

Reply:
You are right. I'm moving it

Reply:
Moved to a new "developer" section of the env file